services/checks/checks.go

```diff
@@ -1092,6 +1093,19 @@ func (s *Service) downloadChecks(ctx context.Context) ([]check.Check, error) {
 
 	resp, err := api.NewRetrievalAPIClient(cc).GetAllChecks(ctx, &api.GetAllChecksRequest{})
 	if err != nil {
+		// If SaaS credentials have become invalid then force a log out so that the next
+		// check download attempt is successfull. Details: https://jira.percona.com/browse/PMM-7965
```

Comment:
This ticket has `internal` security level, nobody outside percona can see it. So better to not include this ticket link. In general, I think better to not include ticket links at all except TODOs.

Reply:
done